plugins/rebalance/eventnotify.go

```diff
@@ -0,0 +1,80 @@
+package rebalance
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+
+	log "github.com/sirupsen/logrus"
+)
+
+//HandleEventNotify updates the rebalinfo in the store with the status sent by
+// the rebalance process
+func HandleEventNotify(status map[string]string) error {
```

Comment:
IIRC, the rebalance process only notifies GD once before it dies. In that case this function should also ensure that the rebalance process is removed from the daemons list. This can be added in a subsequent PR.